Gateway/Validator/CheckoutResponseValidator.php

```diff
@@ -107,15 +108,41 @@ public function validate(array $validationSubject)
                         }
                     }
 
-                    // Save cc_type if available in the response
-                    if (!empty($response['additionalData']['paymentMethod'])) {
-                        $ccType = $this->adyenHelper->getMagentoCreditCartType(
-                            $response['additionalData']['paymentMethod']
+                    // TODO doudle check this
+                if (isset($response['additionalData']) && is_array($response['additionalData'])) {
+                    $additionalData = $response['additionalData'];
+                    if (isset($additionalData['boletobancario.dueDate'])) {
```

Comment:
were these the old fields for boleto in PAL /authorise? I believe you get this stuff elsewhere now: https://docs.adyen.com/payment-methods/boleto-bancario/api-only#make-payment

Reply:
Indeed, we are going to move to action anyway so these will be removed. (I just copied them from the removed validator for now to keep the same code we had)